classes/issue/IssueDAO.inc.php

```diff
@@ -19,53 +19,13 @@
 
 
 class IssueDAO extends DAO implements PKPPubIdPluginDAO {
-	var $caches;
-
-	/**
-	 * Handle a cache miss.
-	 * @param $cache GenericCache
-	 * @param $id string
-	 * @return Issue
-	 */
-	function _cacheMiss($cache, $id) {
-		if ($cache->getCacheId() === 'current') {
-			$issue = $this->getCurrent($id, false);
-		} else {
-			$issue = $this->getByBestId($id, null, false);
-		}
-		$cache->setCache($id, $issue);
-		return $issue;
-	}
```

Comment:
It's worth asking @ajnyga to test out this change on journals.fi. I have a vague recollection of him encountering a big performance issue with a missed cache hit on issues that led to thousands of database requests.

Reply:
not sure if I remember this. But we are still in 3.1.2. I am soon starting to test upgrades to 3.2 with a copy of the production site, so I can test this then.